### DISCUSSION TITLE
Eraser tool doesn't actually erase with Wacom Intuos S stylus

### DISCUSSION THREAD

**Discussion by lphilpot** [xournalpp/xournalpp#3861] [Q&A] [answered]
Xournal++ 1.1.1, Windows 11

Just wondering if anyone has run into this...

My Intuos S pen has no eraser so I wanted to configure it so I could draw by default (tip), pan with button1 and erase with button2. I've gotten the first two working, but despite the eraser square displaying (and the toolbar's icons changing focus, etc.) when I click the second button, it doesn't actually erase anything. No matter how I configure my stylus in Xournal and/or the Wacom control panel the second button won't erase.

Through trial and error I finally found that...

- In the Wacom control panel: setting the stylus to tip = click, button1 = Scroll and button2 = Right click
- In Xournal++ preferences: setting Stylus | Button 1= Tool - don't change, Button 2 = Eraser (and Eraser = whatever, N/A)

...would result in the behavior I described in the first paragraph, i.e., 2 out of 3 functions working. I tried _**lots**_ of other options / combinations but never got any closer and something else always didn't work. So I think (hope?) I'm close. I'm very new to pad/stylus stuff in general so I may be missing something.

Any advice?

Thanks.

**Comment by Conlex** [comment]
Hi, I think it is a bug of version 1.1.1 since I got the same issue both on the desktop and laptop with Wacom Intuos S.
My solution is to go back to a older version like:

Version 1.1.0
Git commit: 9d1277de
Built on: Jul 18 2021, 21:36:30
with libgtk 3.24.29

**Comment by lphilpot** [comment]
I'll give that a shot, thanks.

**Comment by lphilpot** [comment, marked as answer]
OK, I now have it working at least to a point good enough for me. Reverting to 1.1.0 didn't fix it, so I upgraded to the latest 1.1.1 dev version which also didn't automatically fix it. But now it's working with the tip / touch drawing, button one (nearest tip) click scrolling and button two (farthest from tip) erasing. Whether all of these settings are strictly required (probably not), or if a simpler config would work better, I don't know, but this is currently working for me.

**_In Wacom Tablet Properties_**
I have a custom profile for Xournalpp
Pen: 
- Tip = Click
- Button 1 = Scroll
- Button 2 = Right Click

**_In Xournal++ preferences_**
_Input System_
- Wacom Co. Ltd. Intous S Pen stylus (pen) = Pen
- Wacom Co. Ltd. Intous S Eraser (eraser) = Eraser
- WACOM Tablet Pressure Stylus (pen) = Disabled
- [X] Enable drawing outside of window
- [X] Merge button events with stylus tip events

_Stylus_
- Button 1 = Tool - don't change
- Button 2 = Eraser
-- Thickness - don't change
--Standard
- Button 3 = Eraser
--Thickness - don't change
--Standard

I think what did it for me was to enable "Merge button events..."
